tests/failpoints/cases/test_disk_full.rs

```diff
@@ -47,11 +49,22 @@ fn test_disk_full_unallowed_leader_behaviors() {
     let new_last_index = cluster.raft_local_state(1, 1).last_index;
     assert_eq!(old_last_index, new_last_index);
 
-    fail::remove("disk_full_peer_1");
+    // Test Txn Prewrite is not allowed.
```

Comment:
It can be moved into `test_disk_almost_full_allowed_leader_txn_behaviors`. And we can rename `test_disk_almost_full_allowed_leader_txn_behaviors` to `test_disk_almost_full_leader_txn_behaviors`.

Reply:
Done!